PersonDataRequest/PersonDataRequest/Controller/DataViewController.swift

```diff
@@ -94,7 +99,7 @@ class DataViewController: UIViewController {
              }
          case .failure(let error):
              DispatchQueue.main.async {
-                 self.errorAlert(with: error)
+                 self.errorAlert(with: error) { self.queryUserData()}
```

Comment:
Please add [weak self] in completion here as well

Reply:
Added